Inc/usb_host.h

```diff
@@ -1,94 +1,94 @@
-/* USER CODE BEGIN Header */
-/**
-  ******************************************************************************
-  * @file           : usb_host.h
-  * @version        : v1.0_Cube
-  * @brief          : Header for usb_host.c file.
-  ******************************************************************************
-  * @attention
-  *
-  * <h2><center>&copy; Copyright (c) 2020 STMicroelectronics.
-  * All rights reserved.</center></h2>
-  *
-  * This software component is licensed by ST under Ultimate Liberty license
-  * SLA0044, the "License"; You may not use this file except in compliance with
-  * the License. You may obtain a copy of the License at:
-  *                             www.st.com/SLA0044
-  *
-  ******************************************************************************
-  */
-/* USER CODE END Header */
-
-/* Define to prevent recursive inclusion -------------------------------------*/
-#ifndef __USB_HOST__H__
-#define __USB_HOST__H__
-
-#ifdef __cplusplus
- extern "C" {
-#endif
-
-/* Includes ------------------------------------------------------------------*/
-#include "stm32f4xx.h"
-#include "stm32f4xx_hal.h"
-
-/* USER CODE BEGIN INCLUDE */
-
-/* USER CODE END INCLUDE */
-
-/** @addtogroup USBH_OTG_DRIVER
-  * @{
-  */
-
-/** @defgroup USBH_HOST USBH_HOST
-  * @brief Host file for Usb otg low level driver.
-  * @{
-  */
-
-/** @defgroup USBH_HOST_Exported_Variables USBH_HOST_Exported_Variables
-  * @brief Public variables.
-  * @{
-  */
-
-/**
-  * @}
-  */
-
-/** Status of the application. */
-typedef enum {
-  APPLICATION_IDLE = 0,
-  APPLICATION_START,
-  APPLICATION_READY,
-  APPLICATION_DISCONNECT
-}ApplicationTypeDef;
-
-/** @defgroup USBH_HOST_Exported_FunctionsPrototype USBH_HOST_Exported_FunctionsPrototype
-  * @brief Declaration of public functions for Usb host.
-  * @{
-  */
-
-/* Exported functions -------------------------------------------------------*/
-
-/** @brief USB Host initialization function. */
-void MX_USB_HOST_Init(void);
-void MX_USB_HOST_DeInit(void);
```

Comment:
I wrote this function because I have no idea why it doesn't exist. The low level version exists but I think it was static or something. I would recommend restoring this. Besides, IIRC this function needs to be called before jumping to DFU mode to ensure it works consistently.